app/utilities/protocol_importers/protocols_io/v3/protocol_normalizer.rb

```diff
@@ -50,6 +50,24 @@ def normalize_protocol(client_data)
 
           { protocol: normalized_data }
         end
+
+        def normalize_list(client_data)
```

Comment:
In the interface `ProtocolImporters::ProtocolNormalizer` this method is called `normalize_all_protocols`. Decide for one name.